Wabbajack.App.Blazor/Main.razor

```diff
@@ -1,13 +1,14 @@
 ﻿<Fluxor.Blazor.Web.StoreInitializer/>
 
 @using Wabbajack.App.Blazor.Shared
-
-<Router AppAssembly="@GetType().Assembly">
-    <Found Context="routeData">
-        <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)"/>
-    </Found>
-    <NotFound>
-        <h1>Not found</h1>
-        <p>Sorry, there's nothing here.</p>
-    </NotFound>
-</Router>
+<CascadingBlazoredModal>
```

Comment:
I've checked the code for Blazored.Modal and they use `CascadingValue` everywhere instead of just using DI. Using `CascadingParameter` is even more expensive than normal `Parameter` so just inject `IModalService`.

Reply:
The CascadingBlazoredModal is required here otherwise the modal won't render. I don't think we need to be this pedantic over performance as we're not doing anything even remotely close to something where UI performance matters this much.